### DISCUSSION TITLE
Rancher- nerdctl not working in windows 10 version under WSL2

### DISCUSSION THREAD

**Discussion by gaurmukesh** [rancher-sandbox/rancher-desktop#3213] [General]
Hi - I am referring Hello World Example | Rancher Desktop Docs
While running nerdctl build --tag helloworld:v1.0 .
nerdctl not working in windows 10 version under WSL2
Getting below error
FATA[0000] lstat /sys/fs/cgroup/rancher-desktop/run/nerdctl-tmp.3927403874/input.3456756224/Containerfile: no such fileor directory

**Comment by jandubois** [comment]
I have no idea what is happening on your machine. It works fine from Powershell, from plain cmd.exe, and from inside an Ubuntu distro on WSL2. Just tested all 3 scenarios with Rancher Desktop 1.6.0 on Windows 10 Pro.

**Reply by gaurmukesh** [reply]
Thanks. Worked for me with Powershell. Not working with Ubuntu20.04 using nerdctl - "FATA[0000] lstat /sys/fs/cgroup/rancher-desktop/run/nerdctl-tmp.2798224990/input.249136068/Containerfile: no such file or directory"

**Comment by appleparan** [comment]
I have the same issue. (not only `build` command, but also other command like `run`) However, after running the command, I found that `tmp` directory already have existed. I think it is a kind of command order problems, the `tmp` directory wasn't created when it needed.

My system info:
* Windows 10 20H2
* WSL2(Debian Bullseye)
* Running command inside WSL2 (with/without `sudo`)

**Reply by appleparan** [reply]
I gave up `nerdctl` and changed to `docker(moby)`, then it worked. I have a `permission denied` problem with `docker` but fixed it by adding `docker` group described in Troubleshooting Tips in docs.
